project/Build.scala

```diff
@@ -55,6 +57,29 @@ object Build extends Build {
         else
           Some("releases" at nexus + "service/local/staging/deploy/maven2")
       },
+      ReleaseKeys.tagName <<= (version in ThisBuild) map (v => v),
+      ReleaseKeys.releaseProcess := Seq[ReleaseStep](
+        checkSnapshotDependencies,
+        inquireVersions,
+        runClean,
+        runTest,
+        setReleaseVersion,
+        commitReleaseVersion,
+        tagRelease,
+        ReleaseStep(
+          action = { state =>
+            val extracted = Project extract state
```

Comment:
And we cannot call `state.extract`. We need to call `Project.extract(configuration key)`